PCbuild/pcbuild.proj

```diff
@@ -17,7 +17,7 @@
     <FreezeProjects>
       <Platform>$(Platform)</Platform>
       <Platform Condition="$(Platform) == 'ARM'">Win32</Platform>
-      <Platform Condition="$(Platform) == 'ARM64'">x64</Platform>
+      <Platform Condition="$(Platform) == 'ARM64'">ARM64</Platform>
```

Comment:
This platform needs to match the host machine (doing the build), not the target architecture, because these projects are run as part of build.

Maybe there's another property that can be used to check the host platform?